atlasdb-timelock-server/src/test/java/com/palantir/atlasdb/timelock/paxos/PaxosTimeLockServerTest.java

```diff
@@ -77,6 +80,14 @@ public void verifyPaxosResourcesAreRegisteredAfterStartup() throws IOException {
     }
 
     @Test
+    public void verifyHealthCheckIsRegisteredAfterStartup() throws IOException {
+        implementation.onStartup(TIMELOCK_CONFIG);
+        verify(environment.healthChecks(),
+                times(1)).register(startsWith("leader-ping"),
+                Mockito.isA(LeaderPingHealthCheck.class));
```

Comment:
nit: `environment.healthChecks()` and `times(1)` are actually both arguments to `verify`, and the matchers are both arguments to `register`. So I think this would be clearer:

```java
verify(environment.healthChecks(), times(1))
        .register(startsWith("leader-ping"), Mockito.isA(LeaderPingHealthCheck.class));
```

👍 for this test, though. Thanks for adding this.